staubli_tx2-90_support/package.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<package>
+  <name>staubli_tx2-90_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Staubli tx2-90 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Staubli tx2-90 manipulators. This includes the base model,
+      as well as models with extended arm and forearm links.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>tx2-90 - Standard Arm - Nominal Speed</li>
+      <li>tx2-90L - Long Arm - Nominal Speed</li>
+      <li>tx2-90XL - Extra Long Arm - Nominal Speed</li>
+    </ul>
+    <p>
+      Joint limits were taken from broschures, torque limits, and maximum joint velocities
+      are based on the
+      information in the <em>Staubli Robotics - TX series 90 family -
+      Instruction Manual</em> version <em>D28082904C – 01/12/2014</em> and might have changed.
```

Comment:
Are the joint limits not also listed in the instruction manual?

Reply:
I only had the brochure at hand but I now found the time to log into their system and download the instruction manual for tx2-90. Will update it now.

They seem to store and mail passwords unencrypted btw ...

Reply:
> They seem to store and mail passwords unencrypted btw ...

Reply:
I send them a mail refering also to a case with 20k€ fine for such a database ...

More on-topic again, did we agree to set nominal or maximum speed in URDF?

Reply:
I set them to maximum speed for now, rechecked everything with the handbook and referenced it as source

Reply:
I would set to to *Nominal*.

See also https://github.com/ros-industrial/staubli_val3_driver/issues/2#issuecomment-565849458.

Reply:
Done